pubspec.yaml

```
@@ -88,3 +91,7 @@ flutter:
   #
   # For details regarding fonts from package dependencies,
   # see https://flutter.dev/custom-fonts/#from-packages
+  fonts:
+    - family: Inter
+      fonts:
+        - asset: fonts/Inter_Font_VariableAxes_slant_weight.ttf
```

```suggestion
        - asset: fonts/Inter_Font_VariableAxes_Slant_Weight.ttf
```

Also, instead, why can't you use Kebab case?

---

I think its actually better to rename the file with `lowercase_with_underscore` style. Because that's the convention followed for naming source files in dart.
But I'm confused here on what naming convention to follow for assets. As, there are .png files for flutter icons, named in both formats `kebab-case` and `lowercase_with_underscores`, generated by flutter itself.

---

Pick what you want. It makes sense to go with snake case but you are just not consistent with anything in the file.